src/ops.jl

```diff
@@ -335,3 +335,9 @@ Base.Test.approx_full(a::FixedArray) = a
 +{m, n, T}(J::Base.LinAlg.UniformScaling, A::Mat{m,n, T}) = A + J
 -{m, n, T}(A::Mat{m,n, T}, J::Base.LinAlg.UniformScaling) = A + (-J)
 -{m, n, T}(J::Base.LinAlg.UniformScaling, A::Mat{m,n, T}) = J.λ*eye(Mat{m,n,T}) - A
+
+# typed floor, ceil, round, trunc
+Base.floor{T}(::Type{T}, A::FixedArray) = map(x->floor(T, x), A)
+Base.ceil{T}( ::Type{T}, A::FixedArray) = map(x->ceil( T, x), A)
+Base.trunc{T}(::Type{T}, A::FixedArray) = map(x->trunc(T, x), A)
+Base.round{T}(::Type{T}, A::FixedArray) = map(x->round(T, x), A)
```

Comment:
Ah, true, I forgot about 0.4 :smile: (even though that's what my lab still uses for "production").

Still, `map` and anonymous functions offer a so-much-simpler approach to writing code for FSA that perhaps one should create a 0.5 branch? (Or conversely, split off a 0.4 branch?)